JustSaying.Sample.Restaurant.KitchenConsole/Program.cs

```diff
@@ -1,55 +1,90 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
 using Amazon;
 using JustSaying.Sample.Restaurant.Models;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
-using System;
-using System.Threading.Tasks;
+[assembly: SuppressMessage("Reliability", "CA2007:Do not directly await a Task", Justification = "Console application do not have a SynchronizationContext")]
 
 namespace JustSaying.Sample.Restaurant.KitchenConsole
 {
     internal class Program
     {
-        private static async Task Main()
+        public static async Task Main()
         {
             Console.Title = "KitchenConsole";
 
             await new HostBuilder()
+                .ConfigureAppConfiguration((hostContext, config) =>
+                {
+                    config.SetBasePath(Environment.CurrentDirectory);
```

Comment:
This wouldn't be correct if you ran `dotnet` with an absolute path from outside the directory tree. I can't remember the exact name, but I think you want `ApplicationContext.ApplicationBase` or something like that instead.

Reply:
ok. Taking a look

Reply:
You're right. `AppContext.BaseDirectory` allows it to be executed from another path